.github/workflows/build.yml

```diff
@@ -340,3 +369,44 @@ jobs:
       - name: copy binaries to s3
         run: |
           aws s3 sync . s3://apex-net/
+
+  build-rpm:
+    needs: ["go-lint"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/setup-buildx-action@v2
+      - name: Install make
+        run: |
+          sudo apt -y install make
+      - name: Build rpm
+        run: |
+          make rpm
+
+  publish-rpms:
```

Comment:
This should live in the `deploy` workflow. You can then get rid of ` if: github.ref == 'refs/heads/main'`

Reply:
OK. We should probably move publishing binaries to S3 there, as well.  None of these are really `deploy`, though ... maybe a new `publish` workflow?

Reply:
... ah, but uploading to s3 depends on artifacts saved off in a past job in the workflow, so I guess that one can't move.

Reply:
done